src/language-markdown/print-list.js

```diff
@@ -0,0 +1,151 @@
+import { align } from "../document/index.js";
```

Comment:
To make this PR easier to review, let's extract print-list.js‎ and print-children.js in main branch first? And they should put in `print/` directory.